config/application.rb

```diff
@@ -34,3 +34,6 @@ class Application < Rails::Application
     config.generators.system_tests = nil
   end
 end
+
+require 'rails/all'
+require 'csv'
```

Comment:
`require 'rails/all'`は不要です。

`require 'csv'`は一番上に `# require "rails/test_unit/railtie"` の下に移動しましょう。

```suggestion
require 'csv'
```

Reply:
修正してみたのですが、このような形で合っているのでしょうか？？
また、指定の記述はOOより上にないといけないなどの決まりがあるのでしょうか。
ファイルに書いてしまえば同じだと思っていたので、調べてみたところ相応の記事が見つかりアセンでした。

参考になるような記事があれば教えていただきたいです。

Reply:
どこに書いてもよいからという理由でどこにでも書くと，可読性が下がります。
`require` しているところに集めるのが普通でしょう。

Reply:
読む側の視点に立った考え方なのですね！
ありがとうございます！！